src/main/java/graphql/Scalars.java

```diff
@@ -178,4 +187,156 @@ public Object parseLiteral(Object input) {
         }
     });
 
+    public static GraphQLScalarType GraphQLBigInteger = new GraphQLScalarType("BigInteger", "Built-in java.math.BigInteger", new Coercing() {
```

Comment:
Minor detail, but I think there's value here in clarifying that these are non-standard types, assuming it was not a happy accident that GraphQLLong was not described as "Built-in Long".